common/speech/lasr_rasa/assistants/receptionist/data/nlu.yml

```diff
@@ -25,6 +25,8 @@ nlu:
     - [Axel](name)
     - [Charlie](name)
     - [Jane](name)
+    - [Jayne](name)
+    - [Jay](name)
```

Comment:
```suggestion
```

Reply:
These have been added in so that the speech can pickup any of the misheard words for "Jane"